starkit/func_EBV.py

```diff
@@ -0,0 +1,30 @@
+
+from __future__ import print_function
+from astropy.coordinates import SkyCoord
+from dustmaps.sfd import SFDQuery
+import astropy.units as au
+
+def getEBV_effect(rtAs_n,dec_n):
+     '''
+     Generates E(B-V) i.e magnitude of Extinction effect with sky coordinates.
+     parameters
+     ----------
+     rightAscension: angular distance of a particular point measured eastward along equator
+     declination: angular distance of a point north or south of the celestial equator
+     frame:  system of metrics to calculate the magnitude of reddening (here)
```

Comment:
what does 'frame' mean? This does not seem to be a parameter in the code

Reply:
Yes, that makes sense. Will remove it.  Thanks @followthesheep :)

Reply:
Done :+1:

Reply:
@followthesheep , May I know why did you mark the above as outdated?